runtime-watcher/internal/cacertificatehandler/ca_certificate_handler_test.go

```diff
@@ -0,0 +1,131 @@
+package cacertificatehandler_test
+
+import (
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-project/runtime-watcher/skr/internal/cacertificatehandler"
+	"github.com/kyma-project/runtime-watcher/skr/internal/tlstest"
+)
+
+func TestGetCertificatePool1(t *testing.T) {
+	t.Parallel()
+	certPath := "ca.cert"
```

Comment:
Perhaps you could use a temporary file from the OS? In this way we don't have to care about the cleanup.